packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -64,7 +83,7 @@ module.exports = {
     // containing code from all our entry points, and the Webpack runtime.
     filename: 'static/js/bundle.js',
     // In development, we always serve from the root. This makes config easier.
-    publicPath: '/'
+    publicPath: publicPath 
```

Comment:
Let’s discuss more in the issue then so that I can get a better understanding of what you wanted. If you could guide me through your development workflow, what’s running on what port, and how you intend to use `CDN_URL`, etc, that would help.